prow/spyglass/lenses/links/links.go

```diff
@@ -90,22 +93,56 @@ func humanReadableName(name string) string {
 	return strings.Join(words, " ")
 }
 
+// Config is an optional configuration for links lens.
+type Config struct {
+	GCSBrowserPrefix string `json:"gcs_browser_prefix,omitempty"`
+}
+
+func parseConfig(raw json.RawMessage) (*Config, error) {
+	c := &Config{
+		GCSBrowserPrefix: defaultGCSBrowserPrefix,
+	}
+	if len(raw) == 0 {
+		return c, nil
+	}
+	return c, json.Unmarshal(raw, c)
```

Comment:
Agree that tests would be nice... I'm _guessing_ that an empty JSON object (something like `{ }`) or one with an empty-string `gcs_browser_prefix` value would break the clicableLink-generation.

Reply:
Removed this section competely